app/src/main/java/com/android/unio/ui/event/EventCard.kt

```diff
@@ -230,7 +230,9 @@ fun EventCardScaffold(
                 modifier =
                     Modifier.padding(vertical = 1.dp, horizontal = 0.dp)
                         .testTag(EventCardTestTags.EVENT_DATE),
-                text = formatTimestamp(event.date, SimpleDateFormat("dd/MM", Locale.getDefault())),
+                text =
+                    formatTimestamp(
+                        event.startDate, SimpleDateFormat("dd/MM", Locale.getDefault())),
```

Comment:
The SimpleDateFormat("dd/MM", Locale.getDefault())) should really be extracted to a constant in case we ever want to change the date format at some point since it's used in ~6 different places. Do it now if you have time or just add it as a task for the future since it's not urgent

Reply:
As SimpleDateFormat does not work well with being in a static context, I decided to just put the formatting strings in Strings.kt :)